### PR TITLE
{tools}[GCC/10.3.0] BeautifulSoup v4.10.0, FACE v1.1.1, FORD v6.1.6, ... w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/ee4c242c87b66044aea31f77f576c095 for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen2

File: easybuild/easyconfigs/f/FACE/FACE-1.1.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'FACE'
+version = '1.1.1'
+
+homepage = 'https://github.com/szaghi/FACE'
+description = """A Fortran Ansi Colors (and Styles) Environment.
+A KISS pure Fortran Library for easy colorize (and stylize) strings."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```
Comment (branfosj) [inline easybuild/easyconfigs/f/FACE/FACE-1.1.1-GCCcore-10.3.0.eb:10]:
```
$ file FACE/1.1.1-GCCcore-10.3.0/include/face.mod
FACE/1.1.1-GCCcore-10.3.0/include/face.mod: gzip compressed data, from Unix, original size 3041
```

So, this should move to `GCC`.

File: easybuild/easyconfigs/f/forbear/forbear-1.2.0-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'MakeCp'
+
+name = 'forbear'
+version = '1.2.0'
+
+homepage = 'https://github.com/szaghi/forbear'
+description = "A Fortran Library for building and running fancy progress bar"
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```
Comment (branfosj) [inline easybuild/easyconfigs/f/forbear/forbear-1.2.0-GCCcore-10.3.0.eb:9]: This also has `.mod` files and should move to `GCC`.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/FACE/FACE-1.1.1-GCCcore-10.3.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/f/FACE/FACE-1.1.1-GCCcore-10.3.0.eb]: Fixed, had forgotten to push this since I made that change locally a long time ago...

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/62310876142bd18bbd7b9928adacd003 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Pull request merged**